src/osyris/spatial/coordinate_transforms.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Osyris contributors (https://github.com/osyris-project/osyris)
+
+import numpy as np
+from . import utils
+
+
+def change_origin(dataset, new_origin):
```

Comment:
Maybe we should use more common names such as `translate` and `rotate`?

Reply:
That is indeed more intuitive.